design-documents/graph-ql/coverage/CartPromotions.graphqls

```diff
@@ -0,0 +1,53 @@
+type Cart{
+
+    /// Deprecated Feilds
+    applied_coupon: applied_coupon @deprecated(reason: "Use CartPromotions.coupons instead ")                               
+    applied_gift_cards: [AppliedGiftCard] @deprecated(reason: "Use CartPrices.gift_cards instead ") 
+    applied_store_credit: AppliedStoreCredit @deprecated(reason: "Use CartPrices.store_credit instead ") 
+
+    available_payment_methods: [AvailablePaymentMethod]
+    billing_address: BillingCartAddress!
+    email: String
+    items: [CartItemInterface]
+    prices: CartPrices
+    selected_payment_method: SelectedPaymentMethod
+    shipping_addresses: [ShippingCartAddress]!
+}
+
+type CartPrices{
+    applied_taxes: [CartTaxItem]
+    grand_total: Money
+    subtotal_excluding_tax: Money
+    subtotal_including_tax: Money
+    subtotal_with_discount_excluding_tax: Money
+
+    /// Deprecated Fields
+    discount: CartDiscount @deprecated(reason: "Use CartPromotions.discounts instead ") 
+
+    //New Fields
+    discounts: CartPromotions
+    gift_cards: [AppliedGiftCards]
+    store_credit: AppliedStoreCredit
+    
+}
+
+type Discount{
+    amount: Money!
+    label: String!
+}
+
+//New Type
+type CartPromotions{
```

Comment:
Schema of cart and cart item promotions seems inconsistent. Why not to keep applied coupon on cart level and have `discounts: [Discount]` on `CartPrices` level?